README.md

```diff
@@ -117,21 +117,83 @@ This tool **does not** provide any features handled by other offline apps such a
 
 On Linux for now.
 
-Get your dev environment ready:
+1. Get your dev environment ready:
 
-```
-cp server/config/database.cfg.dev server/config/database.cfg
-make dev-environment
-```
+   cp server/config/database.cfg.dev server/config/database.cfg
+   make dev-environment
+
+2. Run the Python backend and React front-end with auto-reload:
+
+   ./run-dev.sh
 
 Whenever you want to reset the database:
 
-```
-make resetdb
-```
+    make resetdb
+
+## Developer Setup on Mac with a Linux VM
+
+1.  Install Parallels.
+
+        brew cask install parallels
+
+2.  Download Ubuntu Desktop `.iso` image file: [ubuntu-18.04.4-desktop-amd64.iso](https://releases.ubuntu.com/18.04.4/ubuntu-18.04.4-desktop-amd64.iso)
+3.  Create Parallels VM using the Ubuntu iso image file.
+4.  Copy ssh public key `~/.ssh/id_rsa.pub` to VM `~/.ssh/authorized_keys`.
+5.  Install NVM (curl is an install dependency)
+    ```
+    sudo apt install curl
+    curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.35.3/install.sh | bash
+    ```
+    Close terminal and reopen.
+    ```
+    nvm install 12
+    ```
+6.  Install `git`
+7.  `git clone` this project on the VM.
+8.  Follow steps above for "Running in Development"
+9.  Setup Github access to allow pushing to repos.
+
+### Optional Steps
+
+1. Install gpg to sign commits.
+2. Setup VS Code to use "Remote - SSH" extension to be able to edit files and access the terminal from within VS Code… and forward ports such that you can use a native browser as well.
+3. Setup dotfiles to your preference.
+
+### Setup SSH Host Shortcut
+
+Set up ssh host shortcut such that you can type `ssh vx` to connect to VM:
+
+1.  Open Settings in the VM (click the caret in the top right to reveal menu to click button to open "Settings")
+2.  Open Network in sidebar.
+3.  Click the gear next to the "Connected" in the "Wired" section.
+4.  Copy the IP address. Eg. `10.211.55.3`
+5.  Back on dev machine, create/open the file `~/.ssh/config` and add:
```

Comment:
I think it's worth using the standard terminology here and using "host" and "guest" (or "VM").